### PR TITLE
Fixed the display pipe power off

### PR DESCRIPTION
**Pull request by pallavigopal**
DPMS off is enough to turn off the display.
with this patch Hotplug works fine.

Jira: None
Test: Hotplug works fine in Android. Tried 10-15 times.
Signed-off-by: Pallavi G <pallavi.g@intel.com>

### CODE REVIEW

**Comment by pallavigopal** [comment]
@kalyankondapally  Please review

File: wsi/drm/drmdisplay.cpp [hunk 1]
```diff
@@ -540,27 +540,10 @@ bool DrmDisplay::GetFence(drmModeAtomicReqPtr property_set,
...
-    }
-
-    ret = drmModeAtomicCommit(gpu_fd_, pset.get(),
-                              DRM_MODE_ATOMIC_ALLOW_MODESET, NULL);
-    if (ret)
-      ETRACE("Failed to disable display:%s\n", PRINTERROR());
-  } else {
-    ETRACE("Failed to allocate property set %d", -ENOMEM);
+  for (const DisplayPlaneState &comp_plane : composition_planes) {
+    DrmPlane *plane = static_cast<DrmPlane *>(comp_plane.plane());
```
Comment (kalyankondapally) [inline wsi/drm/drmdisplay.cpp]: Which Scenario is this fixing hotplug ? External monitor works fine for me on APL and handles hotplugs correctly. While the patch makes sense just wondering what issue is it fixing.

Comment (pallavigopal) [reply wsi/drm/drmdisplay.cpp]: without this fix Hotplug is not working in KBL-RVP. The ddb is set to 0 for the secondary display.
dpms off is enough to disable the display no need to call atomic call with all the properties set to 0.

Comment (pallavigopal) [reply wsi/drm/drmdisplay.cpp]: Even display suspend/resume works fine with this patch.

Comment (kalyankondapally) [reply wsi/drm/drmdisplay.cpp]: I think we can probably skip this whole "disable" part in drmdisplay when monitor is physically disconnected. So, for powermode changes, ya we should still call this and setting DPMS_OFF should be enough. Incase of physical disconnection we dont need to even call this. Probably in Physical Display https://github.com/intel/IA-Hardware-Composer/blob/master/wsi/physicaldisplay.cpp#L115 we can change this to a new entry point in DisplayQueue. That function in DisplayQueue would call HandleExit. HandleExit could take in a boolean to represent if we want to call the disable or not ?

Comment (kalyankondapally) [reply wsi/drm/drmdisplay.cpp]: Merged this patch, can you send a follow up patch with above mentioned change

**Pull request closed without merging**